rust/operator-binary/Cargo.toml

```diff
@@ -11,7 +11,7 @@ version = "0.8.0-nightly"
 [dependencies]
 stackable-kafka-crd = { path = "../crd" }
 stackable-kafka-operator = { path = "../operator" }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.22.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.25.3" }
 
 clap = { version = "3.2.22", features = ["derive", "env"] }
 serde_yaml = "0.8.24"
```

Comment:
serde_yaml is reexported from the operator-rs, can be removed

Reply:
Done.